public/app/features/templating/template_srv.ts

```diff
@@ -243,6 +243,7 @@ export class TemplateSrv implements BaseTemplateSrv {
       case 'date': {
         return this.formatDate(value, args);
       }
+      default:
```

Comment:
maybe a comment and a test?  is raw supported in the 7.2 branch that has a registry?

Reply:
yea, reason for no test is that there is a different fix in master, this is just temporary backport as I missed this default logic in the old templateSrv logic